src/components/toolbar/Toolbar.tsx

```diff
@@ -146,6 +150,8 @@ Toolbar.Item = function ToolbarItem(props: ToolbarItemProps) {
     disabled: itemDisabled,
     noTooltip = false,
     isPopover,
+    noArrow = false,
+    interactionKind,
```

Comment:
The interaction kind should be set on the Toolbar (you already can via `popoverInteractionKind`)

I don't think having different interaction on items of the same toolbar is good UX.

Reply:
Have you tried using `Toolbar.PopoverItem`?

Reply:
yes, it works well but the tooltip not working
![image](https://github.com/zakodium-oss/react-science/assets/35760236/3aa19715-dcb3-4e37-8f13-335a4517d7c9)
![image](https://github.com/zakodium-oss/react-science/assets/35760236/6a774f65-8f85-4bd2-9176-82375c71f099)

Reply:
> Toolbar.PopoverItem

Sorry for the confusion. I initially thought you were referring to `ToolbarPopoverItem`, but actually, I'm using `ToolbarPopoverItem` and `Toolbar.Item` components.

Reply:
`ToolbarPopoverItem` == `Toolbar.PopoverItem`

My suggestion is that you use `Toolbar.PopoverItem` component to render some custom content instead of the simple tooltip of the `Toolbar.Item` component. If something is not working as expected with this component then I can look into it if you provide me with some more detailed explanations of the issue.

On `Toolbar.Item`, the `title` prop should remain `string`. 

You will probably want to add the ability to set or not the little "chevron" on the toolbar button (they make sense from menus, not for the type of popover content you created in your story).